e2e/deployers/appset.go

```diff
@@ -77,6 +77,21 @@ func (a ApplicationSet) Undeploy(ctx types.TestContext) error {
 		return err
 	}
 
+	err = WaitApplicationSetDeleted(ctx)
+	if err != nil {
+		return err
+	}
```

Comment:
Use the standard style - same for all new code.

    if err := ...; err != nil {
       return err
    }

This style:

    err := ...
    if err != nil {

should be used only if we want to use the error outside of the if block.